### ISSUE TITLE
Missing `typeof`  in "index.js"

### ISSUE THREAD

**Issue by jacobbubu** [dominictarr/level-scuttlebutt#10]
Line 132 in "index.js"

`'function' === scuttlebutt.setId` should be `'function' === typeof scuttlebutt.setId`

**Comment by dominictarr** [comment]
you are correct, I've updated it... but it looks like this is broken with the new levelup, etc,
and I don't have time to fix this right now.
